src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -20462,6 +20468,7 @@ s32 func_8085B28C(PlayState* play, Player* this, PlayerCsAction csAction) {
         }
 
         if ((this->actor.id == ACTOR_EN_TEST3) && (csAction < 0)) {
```

Comment:
```suggestion
        if ((this->actor.id == ACTOR_EN_TEST3) && (csAction <= PLAYER_CSACTION_NEG1)) {
```

Reply:
I thought about doing this, but I think it's more important to highlight that any negative number works instead of it being tied to this specific enum, similar to here: https://github.com/zeldaret/mm/blob/23beee0717364de43ca9a82957cc910cf818de90/src/overlays/actors/ovl_player_actor/z_player.c#L5115-L5116

Although it may still be worth using the enum. Thoughts?

Reply:
What about leaving a comment with a brief explanation for it?